cookbooks/travis_build_environment/recipes/git.rb

```diff
@@ -12,10 +12,12 @@
   action %i[install upgrade]
 end
 
-packagecloud_repo 'github/git-lfs' do
-  type 'deb'
-end
+if node['kernel']['machine'] != 'ppc64le'
```

Comment:
I think this `if` condition should be added to each resource as a `not_if` rather than wrapping them in a ruby-level `if`.